_posts/2023-04-10-opensource-perf.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "Improving JSON parsing performance in opensearch-java"
+authors:
+  - chibenwa
+date:   2023-04-10
+categories:
+  - technical-post
+meta_keywords: OpenSearch, Apache James, Flame graph, Performance, Driver
```

Comment:
Keywords are too broad, please update the meta with the following: 

**meta_keywords:** Apache James, flame graph, flamegraph, JSON parsing performance< OpenSearch Java driver